src/extensions/layout-selector/index.js

```diff
@@ -309,20 +315,19 @@ class LayoutSelector extends Component {
 
 		return ! isActive ? null : (
 			<Modal
-				title={ (
-					<Fragment>
-						{ __( 'Add new page', 'coblocks' ) }
-						<span>{ __( 'Pick one of these layouts or start with a blank page', 'coblocks' ) }</span>
-					</Fragment>
-				) }
-				onRequestClose={ () => {
-					this.useEmptyTemplateLayout();
-					closeTemplateSelector();
-				} }
```

Comment:
This title needs to stay for accesibility reasons and the `onRequestClose` is required for the click outside of the modal to default to an empty page.

For the change in sidebar and header treatment, I would use a CSS class to apply similar styles to each while still maintaining them as individual pieces. Think of the title and modal content as two distinct areas each with a sidebar.

Splitting them out into those different areas might make it more difficult for mobile styles (which aren't in a great shape right now).